src/cls/cas/cls_cas_internal.h

```diff
@@ -219,6 +219,7 @@ struct chunk_refs_by_hash_t : public chunk_refs_t::refs_t {
   void bound_encode(size_t& p) const {
     p += 6 + sizeof(uint64_t) + by_hash.size() * (10 + 10);
   }
+#define __le32 ceph_le32
```

Comment:
probably better to just use ``ceph_le32`` in place of ``__le32`` below so we don't mess with the #defines...

Reply:
That would have my preference, but I tried fixing it like in a lot of other header files.
I'll update